locales/ro/ro.json

```diff
@@ -364,9 +364,9 @@
     "Mauritius": "Mauritius",
     "May": "Mai",
     "Mayotte": "Mayotte",
+    "Meeting": "Întalnire",
     "Mexico": "Mexic",
-    "Micronesia, Federated States Of": "Micronezia",
-    "Micronesia, Federated States of": "Micronesia, Federated States of",
```

Comment:
You have deleted the key. So you don't have to do it.

`Micronesia, Federated States Of` !== `Micronesia, Federated States of`